content/docs/how-we-work/coding-principles-standards.md

```diff
@@ -1,82 +1,68 @@
+description: At Axelerant, we write code not only to solve a specific problem but also to provide a reusable solution that can be used at a community level. We have principles that can guide us to the desired code standards.
 ---
 
 # Coding Principles and Standards
 
-At Axelerant, we write code not only to solve a specific problem but also to provide a reusable solution that can be used in a community level. We have principles that can guide us to the desired code standards.
+At Axelerant, we write code not only to solve a specific problem but also to provide a reusable solution that can be used at a community level. We have principles that can guide us to the desired code standards.
 
 ## Coding Principles
 
 - We use the correct layer of abstraction.
 
 - We write clean, simple code and explain our purpose with comments.
 
-- We contribute back everything we can, because it supports us, our clients and the community.
+- We contribute back everything we can because it supports us, our clients, and the community.
 
-- We manage patches in a standard way so that we can know if code/contrib has been changed, how and why. Every change should be contributed back, and the patch name should include the drupal.org issue NID.
+- We manage patches in a standard way so that we can know if code/contrib has been changed and how and why. Every change should be contributed back, and the patch name should include the drupal.org issue NID.
 
 - We write secure code.
 
 - We write code that anticipates change, so that we can implement (most) changes in an easy and Agile way.
 
-- We write API layer that doesn't assume a particular UI, so that the API could be used in multiple (and unpredictable) situations (For example, used in these UIs: Web site, drush, REST, SMS).
+- We write an API layer that doesn't assume a particular UI, so that the API could be used in multiple (and unpredictable) situations (For example, used in these UIs: Web site, Drush, REST, SMS).
 
-- We remove dead code, because it's dead (and because it's confusing to leave it).
+- We remove dead code because it's dead (and because it's confusing to leave it).
 
 - We treat one as a special case of many, so that the logic is in one place and we don't have to maintain it in two places.
 
-- When writing getter functions, consider handling the logic in the many only. A `get_one` can call the `get_many`.
-
-- We use doxygen commenting standards on the top of functions.
+- We use Doxygen commenting standards on the top of functions.
 
 - We include implementation notes and end-user documentation in our tickets.
 
 ## Coding Standards
 
-- We follow established team practices so we can easily work with each others code, and contribute back.
+- We follow established team practices so we can easily work with each other's code and contribute back.
 
-- We don't hard code
+- We don't hardcode.
```

Comment:
This still doesn't help and it feels like there is a word missing. Besides, if I now understand correctly, this is very similar to the line below about putting any text longer than 2 lines in a variable at least.